### DISCUSSION TITLE
Payment status field is not editable in Sale Orders

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#14553] [How to] [answered]
Hello,

I'm working on YetiForceCRM v6.1.0.
The "Payment status" field in Sale Order is readonly. How I can  make it editable?
If it's not possible, how I can change the value of this field?

Regards
Antonio

**Comment by vovpff** [comment, marked as answer]
You need to add related Payments In record to sales order. Status will change automatically

**Reply by antcorro** [reply]
I had already tried to add related Payment In, but nothing happened. Now I have realized that the cron does the job. Thank you!

**Reply by vovpff** [reply]
Payment In should have status Accepted. After batch cron shoot status of invoice should be cjanged automatically

**Comment by antcorro** [comment]
Hello, there is a problem in updating Payment Status of a Sale Order.
I heve this data:
Total net price: 388,00
Total discount: 48,99
Total net price discounted: 339,01
VAT: 74,58
Total price: 413,59 (this is calculated and it's correct, because is 339,01 + 74,58

The problem is: I have added a Payment In in status Paid (accepted) of 413,59, but when the cron update the Payment Status in the Sale Order it put the state UNDERPAID instead of PAID.

I think the problem is the discount...for other Sales Order without discount this doesn't happen

Any suggestion?

Regards
Antonio

**Reply by vovpff** [reply]
I'm not sure but your real values can have more digits after comma like total sum is 100.0001 but you see only 100.00. if you add payment 100.00 you order will be underpaid.

You can change number of decimals in user settings and check.

Whole system mechanisms calculates currency with 8 digits.

My solution is: new field wit percentage of total payments + remastered total calculations with rounding to 2 digits

**Reply by antcorro** [reply]
You are right. The problem is that 339,01 + VAT of 22% = 413,5922 and the sistem does not round the amount.
In the user settings I have 2 decimal number, but the system just shows 2 digits, do not round, so the last "22" are hidden but is used to check the Payment Status. In my opinion this is not correct.
The correct value is 413,59 rounded to 2 digits.
Is it possible to round the amount to 2 digits?

Thank you very much.

Regards
Antonio

**Reply by vovpff** [reply]
You can modify files as you need.

**Reply by mariuszkrzaczkowski** [reply]
Create issue, you only need to document it well to understand it
